wallet/src/ui/app/pages/dapp-tx-approval/DappTxApprovalPage.module.scss

```diff
@@ -1,14 +1,90 @@
-.value {
-    font-size: 13px;
+@use '_values/colors';
+
+.card {
+    background: colors.$gray-40;
+    border-radius: 10px;
+    border-bottom-left-radius: 0;
+    border-bottom-right-radius: 0;
+}
+
+.content {
+    padding: 15px 20px;
+    display: flex;
+    flex-direction: column;
+    gap: 12px;
+}
+
+.row {
+    display: flex;
+    justify-content: space-between;
+    font-size: 14px;
+    line-height: 13px;
+
+    dt {
+        color: colors.$gray-80;
+    }
+
+    dd {
+        color: colors.$gray-90;
+    }
+}
+
+.card::after {
+    content: '';
+    background-image: url('data:image/svg+xml;utf-8,<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 4 1" preserveAspectRatio="none"><polygon style="fill:white;" points="0,1 2,0 4,1"/></svg>');
```

Comment:
Why not use it as a font icon or even save the svg as a file and let webpack load it?

Reply:
It's not really a font icon, but I'll try to use a webpack import here.